openApiDocs/v1.0/Users.yml

```diff
@@ -787,6 +795,152 @@ paths:
         default:
           $ref: '#/components/responses/error'
       x-ms-docs-operation-type: operation
+  '/users/{user-id}/contactFolders/{contactFolder-id}/contacts/{contact-id}/photo/$value':
+    get:
+      tags:
+        - users.user
+      summary: Get media content for user from users
+      operationId: users.user_GetContent
```

Comment:
It looks like DevX API is returning the wrong OpenAPI docs. Stream support should only be in staging (`graphexplorerapi-staging`) and not prod (`graphexplorerapi`).
cc @irvinesunday